eslint.config.mjs

```diff
@@ -0,0 +1,5 @@
+import tsBaseConfig from '@map-colonies/eslint-config/ts-base';
+import jestConfig from '@map-colonies/eslint-config/jest';
+import { config } from '@map-colonies/eslint-config/helpers';
+
+export default config(jestConfig, tsBaseConfig, { ignores: ['drizzle.config.ts'] });
```

Comment:
ignore drizzle but we dont have drizzle in our project, why not also include typeorm ?

Reply:
removed